### PR TITLE
AMP-31 Analytics implementation

### PR DESCRIPTION
**Pull request by dosyara**
Simple page view event with minimum of data.

This is still work in progress, send it to get input on it.
Analytics request we send from AMP pages:
screen shot 2018-06-05 at 15 11 16
screen shot 2018-06-05 at 15 11 22

Main questions for me:
Is the analytics url correct?
Is it enough of data for the first run?
What else we would need to track in the future?

What do you think?

### CODE REVIEW

File: src/app/components/analytics/analytics.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const getAnalyticsData = ({ headline }) => ({
+  __html: JSON.stringify({
+    requests: {
+      base: 'https://${trackingServer}/b/ss/${accounts}/1/AMP-0.1/s${random}',
+      pageView:
+        '${base}?AQB=1&vid=CLIENT_ID(adobe_amp_id)&pageName=${pageName}&j=amp&AQE=1',
+    },
+    vars: {
+      trackingServer: 'sstats.economist.com',
+      accounts: 'economistcomamp',
```
Comment (dosyara) [inline src/app/components/analytics/analytics.js]: `economistcomamp` - is my invention. Not sure if this correct. On e.com we have `economistcomprod` and `economistcomdev`.

File: src/app/components/analytics/analytics.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      },
+    },
+  }),
+});
+
+const Analytics = ({ data: { article: { headline } = {} } }) => (
+  <amp-analytics type="adobeanalytics">
+    <script
+      type="application/json"
+      dangerouslySetInnerHTML={getAnalyticsData({ headline })}
```
Comment (dosyara) [inline src/app/components/analytics/analytics.js]: Did not find better way to print json object, but with `dangerouslySetInnerHTML`. Any suggestions?

Comment (dosyara) [reply src/app/components/analytics/analytics.js]: it did not work for me, i got escaped json

Comment (dosyara) [reply src/app/components/analytics/analytics.js]: right! thank you!

File: src/app/components/analytics/analytics.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+const getPropTitle = ({ contentType, headline }) => {
+  let propTitle;
+  if (contentType === TYPE_BLOG) {
+    propTitle = `blog|${slug(headline)}`;
+  } else {
+    propTitle = slug(headline);
+  }
+  return propTitle;
+};
+function articlePublishDate(date) {
```
Comment (wajerrr) [inline src/app/components/analytics/analytics.js]: `const articlePublishDate = (date) =>`

File: src/server/utils/user.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+const getUserId = (state = {}) => {
+  let userId = state.ec_uid;
+  if (!userId) {
+    const userCookie = state['Econ.user.user'];
+    if (userCookie) {
+      try {
+        userId = JSON.parse(decodeURIComponent(userCookie)).uid;
+      } catch (e) {
+        // can't parse cookie
```
Comment (yasgreen93) [inline src/server/utils/user.js]: No error here?

**Pull request merged**